packages/traceability/issuer-irec-api-wrapper/src/Device.ts

```diff
@@ -29,11 +29,11 @@ export class DeviceCreateParams extends FileIds {
 
     @Expose({ name: 'device_type_code', toPlainOnly: true })
     @IsString()
-    deviceType: string;
+    fuelType: string;
```

Comment:
Mismatch between `@Expose({ name: 'device_type_code', toPlainOnly: true })`  and `fuelType: string`.
Same goes for line 36, line 99, line 104 in this file.

Reply:
it is not a mismatch. IREC API expects values with "TC" prefixes here. And we want "TC" to be a fuel type. So I changed it since we want to use different from IREC terminology

Reply:
see here
https://evident-registry.atlassian.net/wiki/spaces/EV/pages/219447320/POST%2Bapi%2Birec%2Bv1%2Bdevice-management%2Bcreate